### DISCUSSION TITLE
Automatically saving jumplist entries for count based line navigation

### DISCUSSION THREAD

**Discussion by rkshthrmsh** [helix-editor/helix#6843] [Ideas]
Quite often, I use count-based navigation to get around the code (with relative numbering enabled). For example, `10gj` or `10gk` to jump 10 lines down and up. I find that going back to the piece of code I was previously working on is not possible using the jumplist since these motions do not save to jumplist automatically. Is it worth adding this feature? Or is there a better way of achieveing this on Helix?

Essentially, a way to jump a few lines around the code and return to the base position.

**Comment by rkshthrmsh** [comment]
#5741 mentions a similar issue for absolute line number based navigation, where`10gg` saves to jumplist and goes to line 10.

**Comment by the-mikedavis** [comment]
You can save your position in the jumplist with `C-s` in normal mode and then jump back to it later. Saving automatically when `gj`/`gk` have a count seems too granular to mee: we don't want to clutter the jumplist with unnecessary points.
